__tests__/scripts.js

```diff
@@ -4,15 +4,63 @@ const assert = require('yeoman-assert');
 const helpers = require('yeoman-test');
 
 describe('generator-one-base:scripts', () => {
-  beforeAll(() => {
-    return helpers.run(path.join(__dirname, '../generators/scripts'))
-      .withPrompts({ someAnswer: true });
+  describe('default', () => {
+    beforeAll(() => {
+      return helpers.run(path.join(__dirname, '../generators/scripts'));
+    });
+
+    it('creates dir structure', () => {
+      assert.file([
+        'src/scripts/modules/.gitkeep'
+      ]);
+    });
+
+    it('creates main.js', () => {
+      assert.file([
+        'src/scripts/main.js'
+      ]);
+    });
+  });
+
+  describe('jquery and lodash enabled', () => {
+    beforeAll(() => {
+      jest.setTimeout(60000);
```

Comment:
Maybe I'm alone on this, but I wish there was another way we could test this without needing to actually wait for yarn to install all packages on every test run. There could be issues with this installation that are outside of the control of the generator, and I'd really like running tests to be super duper fast so people (me) are encouraged to run them more often.

Reply:
yeah, this is really annoying. I want to make sure the proper modules are installed, but it takes like 20-40 sec for this test. I was hoping maybe we could jack into the overall yarn list without needing to actually install them.

Reply:
It'd be great if we could just expect `installDependencies` to have been fired with `[x, y]` in the list or something like that.

Reply:
yeah, I think that's good. testing that the `package.json` gets written to is enough for me (and what I was getting at in the other PR), but I'm not sure how to do that since our package.json doesn't get written to until yeoman installs stuff. 

We could just write to the package.json directly and read that, the downside is then we don't get version info.

I also feel like if we don't care what gets installed in the styles generator, we shouldn't care here.

Reply:
Possible to just test that `this.props.deps` contains `lodash`? I'm not super familiar with testing in Yeoman.